apps/mark-scan/backend/src/custom-paper-handler/cli/state_machine_cli.ts

```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore next */
```

Comment:
This is a dev tool that might be removed soon, so not investing in tests right now.